indra/newview/llvoavatar.cpp

```diff
@@ -11292,7 +11293,10 @@ void LLVOAvatar::setOverallAppearanceNormal()
 		return;
 
     LLVector3 pelvis_pos = getJoint("mPelvis")->getPosition();
-	resetSkeleton(false);
+    if (isControlAvatar() || mLastProcessedAppearance)
```

Comment:
This check makes no sense. control avatars won't get here.

Reply:
This check makes sense just because without it we receive many stupid warnings from resetSceleton()

Reply:
There is a isControlAvatar() check at string 11292, checking it here makes no sense.

Reply:
I just try to preserve the parity in the logics

LLControlAvatar is inherited of LLVOAvatar,
and technically it is possible to call setOverallAppearanceNormal() from this class

It is unsafe practice to make some assumptions of impossibility some conditions
We'd better leave this check here